index.d.ts

```diff
@@ -0,0 +1,9 @@
+declare namespace anonymizeIP { }
```

Comment:
Thanks for your contribution! I'm not very familiar with TypeScript, so I'm wondering why the namespace declaration is necessary here, if the name "anonymizeIP" is declared as a function on line 3.

Reply:
Had some trouble with this myself. This is to deal with the function being exported for use as a CommonJS module. There's an [example function module](http://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-function-d-ts.html) in the TS docs. Since no other variables are exported, the namespace is blank.

Reply:
It's also a workaround for supporting importing ip-anonymize as an ES6 module. Both of the following will work with the blank namespace.
```ts
const anonymizeIP = require('ip-anonymize')
import * as anonymizeIP from 'ip-anonymize'
```

Reply:
Thanks for clarifying that, and for your contribution! I'll merge and publish an updated version to NPM today.